eng/Versions.props

```diff
@@ -2,13 +2,13 @@
 <Project>
```

Comment:
Do we have to tell it to not produce stable versions somewhere?

Reply:
No, we never produce a stable package from coreclr